src/composer.ts

```diff
@@ -247,19 +248,19 @@ export class AtomicTransactionComposer {
     /** The number of extra pages to allocate for the application's programs. Only set this if this is an application creation call. If omitted, defaults to 0. */
     extraPages?: number;
     /** Array of Address strings that represent external accounts supplied to this application. If accounts are provided here, the accounts specified in the method args will appear after these. */
-    appAccounts?: string[];
+    appAccounts?: Array<string | Address>;
     /** Array of App ID numbers that represent external apps supplied to this application. If apps are provided here, the apps specified in the method args will appear after these. */
-    appForeignApps?: number[];
+    appForeignApps?: Array<number | bigint>;
```

Comment:
I think the goal is Postel's Law: "be conservative in what you send, be liberal in what you accept". You want callers to be able to give 10283 as an app id, even though the class stores 10283n either way.  I'm on board for that, just wondering if it could be tidier.

Reply:
I'm ok as is, but have a suggestion in case you like it.  The type alias for `number | bigint` could be `uint64`. That expresses _why_ we're taking both.

Up to you, feel free to resolve after reading, either way.

Reply:
After more thought I've decided to stay with the union type here. It's the clearest and simplest solution.

The `uint64` idea is nice in that it does capture the meaning behind the union type., but I fear it would accidentally end up in internal fields or output types. That would not be desirable because a union type there can only add complexity and confusion.